File: Resource/QLNH-Freelance/ComNha/ComNha/Models/Khachhang.cs
namespace ComNha.Models
{
    public class Khachhang
    {
        public int MaKhachHang { get; set; }
        public string SoDienThoai { get; set; } = null!;
        public string TenKhachHang { get; set; } = null!;
        public string? DiaChi { get; set; }
        public string? Email { get; set; }
    }
}

File: References/restaurent-management-comnha/src/main/java/tdtu/thinh6/restaurent_management_comnha/domain/Khachhang.java
package tdtu.thinh6.restaurent_management_comnha.domain;

import java.util.Set;
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.OneToMany;
import lombok.Getter;
import lombok.Setter;


@Entity
@Getter
@Setter
public class Khachhang {

    @Id
    @Column(nullable = false, updatable = false)
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer maKhachHang;

    @Column(nullable = false, length = 20)
    private String soDienThoai;

    @Column(nullable = false, length = 50)
    private String tenKhachHang;

    @Column(length = 50)
    private String diaChi;

    @Column(length = 50)
    private String email;

    @OneToMany(mappedBy = "maKhachHang")
    private Set<Hoadon> maKhachHangHoadons;

    @OneToMany(mappedBy = "maKhachHang")
    private Set<Lichhen> maKhachHangLichhens;

}
